### PR TITLE
Bug fixes

### LINKED ISSUE #26: HouseForm required attributes not working properly
- Submit Form should be reverted to:

```
const submitForm = (event) => {
        const check = checkValidity()
        if(input.houseName && check) {
            event.preventDefault();
            const data = {
                name: input.houseName,
                members: input.membersNames
            }
            console.log(data)
            clearForm();
        }
    }
```

- And the submit button will need to live inside of the Form element but this presents some styling challenges
- maybe removing the see schedule button and auto navigating the user will fix multiple issues at once!

### LINKED ISSUE #28: Bring the Monday button into autofocus when the user navigates to that page
I think I can help with this! I did something similar with What's Cookin using the` .focus()` method

### PR DESCRIPTION
**Pull request by Kanderson58**
# Description
- Put submit buttons back into forms
- Autofocus on Monday when dashboard loads
- Put event.preventDault() back into if/else statement

## Type Of Change

- [x]  Bug fix
- [ ]  New feature
- [ ]  Refactoring
- [ ]  Testing

# How Has This Been Tested?

- [x]  local host
- [x]  dev tools
- [ ]  cypress

# Related Issues:

- [x]  closes  #28 closes #26 

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: src/Components/Dashboard/Dashboard.js
Comment (LeftyLincoln) [inline src/Components/Dashboard/Dashboard.js:1]: I didn't know about the autofocus, this is great. Nice work!

**Review by LeftyLincoln — APPROVED** [review]
This request looks like it cleans up a few of our bugs and starts our cypress build out. Great work!

File: src/Components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -80,15 +80,18 @@ const Dashboard = () => {
...
   const weekButtons = daysOfWeek.map(day => <button key={day} className={`day-btn ${day}`} onClick={() => setDayOfWeek(day)}>{day}</button>)
 
   return (
     <div className="dashboard">
       <div>
         <h1>{householdName}'s Chore Schedule</h1>
 
-      <div className="week-nav">{weekButtons}</div>
+      <div className="week-nav">
+        <button className="day-btn Monday" onClick={() => setDayOfWeek("Monday")} autoFocus>Monday</button>
```
Comment (qrispi) [inline src/Components/Dashboard/Dashboard.js:92]: Love how easy this fix was! Thanks for sharing autoFocus with us!

**Review by qrispi — APPROVED** [review]
LGTM!

**Pull request merged**